projects/canopy/src/lib/message-with-brand-icon/message-with-brand-icon.component.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../styles/mixins';
+
+.lg-message-with-brand-icon {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  &__icon {
+    width: var(--brand-icon-xxs) !important;
```

Comment:
The `!important` here is needed to override the brand icon size.

Reply:
🤮

Reply:
at first I was going to ask... "could you not set it implicitly in the html rather than via the css?"

...then I saw the breakpoint...

which is interesting as I guess this is a scenario that might occur quite a lot with brand-icons, and is maybe something that the component itself could be extended towards, similar to the #362 proposal.

Reply:
I guess the brand icons could simply always be `xxs` on `sm` breakpoint but that would be a breaking change.
I'll check with James.

@andy-polhill do we think we could merge this bit of code in the meantime as it's blocking one of my project stories? In that case I'll make sure to create a new PR after discussing the options with the designers.

Reply:
I think thats fine as it could balloon otherwise, I might convert this thread to a new issue though.